prove.go

```diff
@@ -473,3 +473,78 @@ func proofAfterDeletion(numLeaves uint64, proof Proof) ([]Hash, Proof) {
 
 	return targetHashes, Proof{proveTargets, hashes}
 }
+
+// GetMissingPositions returns the positions missing in the proof to proof the desiredTargets.
+//
+// The proof being passed in MUST be a valid proof. No validity checks are done so the caller
+// must make sure the proof is valid.
+//
+// The passed in desiredTargets also MUST be a valid position in the accumulator. There are
+// no checks to make sure the desiredTargets exist in the accumulator so the caller must
+// check that they indeed do exist.
+func GetMissingPositions(numLeaves uint64, proof Proof, desiredTargets []uint64) []uint64 {
```

Comment:
Since you're not using proof.Hashes, could be cleaner to use:
```
func GetMissingPositions(numLeaves uint64, proofTargets, desiredTargets []uint64) (neededPositions []uint64) {
```

desired works ok for "wanted" but doesn't seem the same as needed.  Like your desiredTargets means here's everything I want to have, and it should return "neededPositions" because that's the stuff you want and don't have yet.,

Reply:
> desired works ok for "wanted" but doesn't seem the same as needed. Like your desiredTargets means here's everything I want to have, and it should return "neededPositions" because that's the stuff you want and don't have yet.,

So `GetNeededPositions()` instead?